page-home.php

```diff
@@ -7,25 +7,104 @@
 
 <?php get_template_part( 'breadcrumb' ); ?>
 
-<div id="primary" class="content-area latin-home">
+<div id="primary" class="content-area latin-home level-one">
 	<div class="container">
-		<div class="row">
-			<main id="main" class="col-xs-12 col-sm-8" role="main">
+        <div class="row" role="banner">
+            <div class="col-md-12">
 				<?php
-				while ( have_posts() ) : the_post();
-					get_template_part( 'content' );
-				endwhile;
+				global $post;
+				$image      = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'full-page-width', true ) );
+				$title      = get_the_title();
+				get_page_banner( 'level one', $title, $image, '', '', '' );
 				?>
-			</main>
+            </div>
+        </div>
+        <div class="row">
```

Comment:
We are missing the main element. Wrap the follow two rows with <main>

Reply:
This has been done